charts/portworx/templates/portworx-controller.yaml

```diff
@@ -120,6 +120,8 @@ spec:
           requests:
             cpu: 200m
       hostNetwork: true
+      tolerations:
+{{ toYaml .Values.tolerations | indent 8 }}
```

Comment:
wouldn't this cause an invalid yaml template if tolerations are empty in the values.yaml. In other words, this will get rendered as

```
  hostNetwork: true
  tolerations:
  affintiy:
    nodeAffinity:
    ...
```

Reply:
Good catch. In our setup we always need tolerations so I didn't test without it. 

To keep the pattern, I copied the code over from the main daemon set, here:
https://github.com/portworx/helm/blob/master/charts/portworx/templates/portworx-ds.yaml#L57

So that probably needs improved as well. I'll update the PR.